.github/env/global.env

```diff
@@ -1,5 +1,5 @@
-      DAPR_CLI_VERSION: 1.11.0
-      DAPR_RUNTIME_VERSION: 1.11.0
+      DAPR_CLI_VERSION: 1.12.0-rc.1
+      DAPR_RUNTIME_VERSION: 1.12.0-rc.3
```

Comment:
This will need to change before merging right?

Reply:
Not now ... it will be changed later finally when release is done.